index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+	<link href='https://fonts.googleapis.com/css?family=Raleway:300,400,600' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<nav>
+		<div class="title">
+			<h1>Downtown Guitars</h1>
+			<h3>Where guitars are downtown...</h3>
+		</div>
+	</nav>
+
+	<article class="jazzmaster">
+		<header>
+			<h1>Fender Jazzmaster</h1>
+		</header>
+
+		<section>
+			<img src="../Jazzmaster-500x500.jpg" alt="Fender Jazzmaster">
+			<p class="copy">Kogi trust fund post-ironic, before they sold out offal pitchfork ugh hammock mixtape flexitarian brunch butcher cred microdosing banjo. Tofu bitters cronut, deep v flexitarian pour-over intelligentsia vice distillery blue bottle. Sartorial farm-to-table art party, single-origin coffee pour-over dreamcatcher health goth swag paleo narwhal. Pabst cornhole drinking vinegar +1 tacos stumptown.</p>
+			<div class="availability">
+				<h4>Available</h4>
+			</div>
+		</section>
+		<section class="specs">
+			<header>
+				<h3>Specifications</h3>
+			</header>
+			<p>Size: 20" x 44"  </p>
+			<p>Weight: 12lbs</p>
```

Comment:
Style requirements ask for the details (the p tags) of the specs and pricing to be bold, not including the title.